core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/ConceptSearchRequestEvaluator.java

```diff
@@ -113,4 +116,17 @@ public Concepts evaluate(CodeSystemURI uri, BranchContext context, Options searc
 		}
 	};
 	
+	/**
+	 * Support value set clause evaluation
+	 * @since 7.7
+	 */
+	public static Collection<String> getIds(Collection<String> querys) {
+		Collection<String> ids = Lists.newArrayList();
+		querys.forEach(query -> {
+			String[] id = query.replaceAll("\\|", "").split(" ", 2);
+			ids.add(id[0]);
```

Comment:
Isn't there a need here to check array length? I'm also not sure whether it is guaranteed that the query string does have at least one whitespace or not?

Reply:
Fixed.